rclcpp/test/topic_statistics/test_subscription_topic_statistics.cpp

```diff
@@ -147,6 +177,38 @@ class EmptySubscriber : public rclcpp::Node
   rclcpp::Subscription<Empty>::SharedPtr subscription_;
 };
 
+/**
+ * MessageWithHeader subscriber node: used to create subscriber topic statistics requirements
+ */
+class DummySubscriber : public rclcpp::Node
+{
+public:
+  DummySubscriber(const std::string & name, const std::string & topic)
+  : Node(name)
+  {
+    // manually enable topic statistics via options
+    auto options = rclcpp::SubscriptionOptions();
+    options.topic_stats_options.state = rclcpp::TopicStatisticsState::Enable;
+
+    auto callback = [this](MessageWithHeader::UniquePtr msg) {
+        this->receive_message(*msg);
```

Comment:
I guess we can remove the method `receive_message` and just pass an empty lambda.